src/rpc.rs

```diff
@@ -157,20 +157,37 @@ mod tests {
         static ref LOCALHOST: SocketAddr = SocketAddr::V4(SocketAddrV4::new(Ipv4Addr::LOCALHOST, 0));
     }
 
+    mod error {
+        lazy_static::lazy_static! {
+            pub static ref CONTRACT_NOT_FOUND: (i64, String) = (-32020, "Contract not found".to_owned());
+            pub static ref INVALID_SELECTOR: (i64, String) = (-32021, "Invalid message selector".to_owned());
+            pub static ref INVALID_CALL_DATA: (i64, String) = (-32022, "Invalid call data".to_owned());
+            pub static ref INVALID_KEY: (i64, String) = (-32023, "Invalid storage key".to_owned());
+            pub static ref INVALID_BLOCK_HASH: (i64, String) = (-32024, "Invalid block hash".to_owned());
+            pub static ref INVALID_TX_HASH: (i64, String) = (-32025, "Invalid transaction hash".to_owned());
+            pub static ref INVALID_BLOCK_NUMBER: (i64, String) = (-32025, "Invalid block number".to_owned());
+        }
+    }
```

Comment:
Should these not just be normal `const`? If you make it `(i64, &str)`?

Reply:
Good point, unfortunately as I wanted to solve the issue of returning a ref to a local var in `get_err` I decided to go straight to an owned type (ie. `String`).  At some point I even gave up with `Cow<'static, str>` as it turned out to be even more characters in those asserts that follow.

Reply:
Ah I missed the fact that it's test infrastructure -- then I don't mind so much, especially if its simpler.